src/queueserver.ts

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
 var axiosInstance = axios.create({
-    baseURL: process.env.REACT_APP_API_PREFIX,
+    baseURL: process.env.REACT_APP_QUEUE_API_PREFIX,
```

Comment:
If you changed the .env file it wasn't committed, and all API calls are failing for me on this branch.

Reply:
updated .env file to fix this: https://github.com/bluesky/bluesky-webclient/pull/21/commits/2657b6de1dca8bb054ef5facddf5660846de506f